### PR TITLE
LPS-195893 Fix 'select all' multiple modifications with more than 100 elements

### PR DESCRIPTION
**Pull request by robertoDiaz**

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

File: modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/util/comparator/UserNotificationEventUserNotificationEventIdComparator.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+package com.liferay.notifications.web.internal.util.comparator;
+
+import com.liferay.portal.kernel.model.UserNotificationEvent;
+import com.liferay.portal.kernel.util.OrderByComparator;
+
+/**
+ * @author Roberto Díaz
+ */
+public class UserNotificationEventUserNotificationEventIdComparator
```
Comment (adolfopa) [inline modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/util/comparator/UserNotificationEventUserNotificationEventIdComparator.java:14]: Shouldn't this be named `UserNotificationEventUserNotificationEventIdOrderByComparator`? We almost always have to include the base class/iface as a suffix in class names.

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/util/comparator/UserNotificationEventUserNotificationEventIdComparator.java (hunk 1 above)
Comment (robertoDiaz) [reply modules/apps/notifications/notifications-web/src/main/java/com/liferay/notifications/web/internal/util/comparator/UserNotificationEventUserNotificationEventIdComparator.java:14]: Sure, it's a typo.

I'll make the class name larger 😃

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by austinchiang** [comment]
Checking failures

**Comment by austinchiang** [comment]
Failures unrelated

**Comment by austinchiang** [comment]
ci:forward:force

**Pull request closed without merging**